sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala

```diff
@@ -1776,4 +1776,33 @@ class SQLQuerySuite extends QueryTest with SQLTestUtils with TestHiveSingleton {
         |FROM (SELECT '{"f1": "value1", "f2": 12}' json, 'hello' as str) test
       """.stripMargin), Row("value1", "12", BigDecimal("3.14"), "hello"))
   }
+
+  test("multi-insert with lateral view") {
```

Comment:
at that time, we haven't had a clear plan to remove hive dependency, and `multi-insert` is considered as a hive only feature. Although the parse rule is in catalyst module, I thought we will separate it into hive module.